src/renderer/src/lib/locale/en/common.json

```diff
@@ -27,9 +30,9 @@
 	"install": "Install",
 	"installFromYourDevice": "Install from your device",
 	"installHapp": "Install Happ",
-	"installing": "Installing...",
 	"installingAppStore": "Installing App Store...",
 	"installingDevHub": "Installing Dev Hub...",
+	"installsDevHub": "Installs DevHub (235.6MB) + Data for Synchronized apps",
```

Comment:
This text is wrong. Devhub is currently only 14MB and it is part of the executable anyway so there is no added disk space from installing alone. Maybe we can just remove that line for now, the modal that pops up is already quite informative.

Reply:
fixed